### ISSUE TITLE
Unable to view followers/friends profiles with VoiceOver on

### ISSUE THREAD

**Issue by KevGP** [metabolist/metatext#185]
Hey,
I'm a blind VoiceOver user of MetaText on iOS 16.1. More often than not, when viewing my followers and friends, if I want to visit the particular profile I need to turn VoiceOver off, tap the area I remember their name to be on-screen, and then turn VoiceOver back on.
Double-tapping the names in the list doesn't do anything most of the time, once it took me to their external website not on Mastodon instead of opening the profile, and a couple times it worked as intended. I suggest maybe add a "view profile" option in the rotor actions if you can't get the profiles to display on double-tap?

**Comment by TheQuinbox** [comment]
Hi,
I reported this in a previous issue, but that's good to know that its voiceover-specific. Going to go ahead and close mine.

**Comment by StephenRadachy** [comment]
Did a bit of digging on this. No where near solving, but wanted to give an update in case someone else decides to pick this up.

There's something funky happening with the `AccountTableViewCell` / `viewModel.select`. For some reason, it intermittently will not call the `didSelectRowAt` tableview function. I've also seen it open the first url in a safari view controller when double tapping with voice over.
